modules/signals/spec/signal-state.spec.ts

```diff
@@ -45,8 +45,8 @@ describe('signalState', () => {
     expect(isSignal(state.ngrx)).toBe(true);
   });
 
-  it('caches previously created signals', () => {
-    jest.spyOn(angular, 'computed');
+  it.skip('caches previously created signals', () => {
```

Comment:
Is this intended?

Reply:
Yes, it throws an error that `computed` cannot be redefined which needs more investigation

Reply:
Try:

```ts
// top level
import { computed } from '@angular/core';

vi.mock('@angular/core', { spy: true });

it('...', () => {
  // ...
  expect(computed).toHaveBeenCalled...
});
```

Reply:
Got that working, thanks!